app/presenters/hyrax/file_set_presenter.rb

```diff
@@ -20,7 +20,7 @@ def initialize(solr_document, current_ability, request = nil)
 
     # CurationConcern methods
     delegate :stringify_keys, :human_readable_type, :collection?, :image?, :video?,
-             :audio?, :pdf?, :office_document?, :representative_id, :to_s, to: :solr_document
+             :audio?, :pdf?, :office_document?, :representative_id, :to_s,  to: :solr_document
```

Comment:
can revert the change to this file (cuts down on the likelihood of merge conflicts)